ofxsMaskMix.h

```diff
@@ -192,16 +192,16 @@ ofxsUnPremult(const PIX *srcPix,
         unpPix[0] = 0.f;
         unpPix[1] = 0.f;
         unpPix[2] = 0.f;
-        unpPix[3] = srcPix[0] / (float)maxValue;
```

Comment:
the culprid was probably line 213 below, which was a real bug. Which OFX host are you using?